CHANGES.md

```diff
@@ -1,6 +1,35 @@
 Libplanet changelog
 ===================
 
+Version 5.1.1
+-------------
+
+To be released.
+
+### Deprecated APIs
+
+### Backward-incompatible API changes
+
+### Backward-incompatible network protocol changes
+
+### Backward-incompatible storage format changes
+
+### Added APIs
+
+### Behavioral changes
+
+### Bug fixes
+
+ -  Fixed an issue with the evidence hash changing depending on the version.
+    [[#3874]]
+
+### Dependencies
+
+### CLI tools
+
```

Comment:
See https://github.com/planetarium/libplanet/pull/3865/files for reference.
We don't categorize changes for minor versions (as they are mostly supposed to be bug fixes).

Reply:
amended https://github.com/planetarium/libplanet/pull/3874/commits/02b3865507374ddc0eff61e3c7b446eabe62f1b5